config/locales/en.yml

```diff
@@ -28,6 +28,27 @@ en:
           label: "No"
         not_sure:
           label: "Not sure"
+    medical_conditions:
+      title: Do you have a medical condition that makes you vulnerable to coronavirus?
+      description: |
+        You’re vulnerable if you:
```

Comment:
Need to add the paragraph text class in to make sure there's spacing after the paragraph and before the list:

```suggestion
        <p class="govuk-body">You’re vulnerable if you:</p>
```